src/Connection.php

```diff
@@ -1417,7 +1417,7 @@ private function _bindTypedValues(DriverStatement $stmt, array $params, array $t
      *
      * @throws Exception
      */
-    private function getBindingInfo($value, $type): array
+    private function getBindingInfo(mixed $value, int|string|Type|null $type): array
```

Comment:
With the native parameter types added, the types specified in phpDoc look redundant. What should we do about them?

Reply:
> With the native parameter types added, the types specified in phpDoc look redundant.

CodeSniffer already removed them if they're clearly redundant. But as soon as there's a human-readable description of the parameter or return value, CodeSniffer cannot decide that.

In this particular case, I'd remove the `@param` annotation for `$value` and keep the one for `$type`.

> Does phpDoc consider the fact that the type annotation may be no longer necessary?

Yes. We can leave out `@param` annotations if the native type declaration is already specific enough for us.

Please, let's remove all annotations that don't provide additional value. But we might need to review most of them manually.